src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Strings.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Strings
+    {
+        private const int DataSize = 100_000;
+
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+
+        private string[] _stringArrayValues;
+        private byte[][] _stringArrayValuesUtf8;
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        public enum Escape
+        {
+            AllEscaped,
+            OneEscaped,
+            NoneEscaped
+        }
+
+        [Params(Escape.AllEscaped, Escape.OneEscaped, Escape.NoneEscaped)]
+        public Escape Escaped;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _arrayBufferWriter = new ArrayBufferWriter<byte>();
+
+            _stringArrayValues = new string[DataSize];
+            _stringArrayValuesUtf8 = new byte[DataSize][];
+
+            for (int i = 0; i < DataSize; i++)
+            {
+                _stringArrayValues[i] = GetString(5, 100, Escaped);
+                _stringArrayValuesUtf8[i] = Encoding.UTF8.GetBytes(_stringArrayValues[i]);
+            }
+        }
+
+        private static string GetString(int minLength, int maxLength, Escape escape)
+        {
+            var random = new Random(42);
```

Comment:
This is going to return you the same string every time, since you're seeding it the same way. Moving this into a static on the class should fix it.

Reply:
Oh, oops, you are right. I had it as a static before, but moved it in. Good catch!

Reply:
This is also duplicated all over the new classes/benchmarks.